carbon-design-system/carbon
# fix(overflowmenuitem): show title tooltip when disabled

Closes #12257 

#### Changelog

**Changed**

- remove `pointer-events: none`, replace with `cursor: not-allowed`
- update story to include an example using `disabled` and `requireTitle`

#### Testing / Reviewing

1. Open the OverflowMenu/Default story
1. hover over the disabled option
1. the title text should appear on hover


https://user-images.githubusercontent.com/3360588/194571257-5ae09b53-98d9-4f99-8981-613101dda88c.mov
